garglk/CMakeLists.txt

```diff
@@ -220,6 +223,10 @@ elseif(UNIX)
     configure_file("garglk.pc.in" "${CMAKE_CURRENT_BINARY_DIR}/garglk.pc" @ONLY)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/garglk.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 
+    if(INSTALL_GARGLKINI)
+        install(FILES "garglk.ini" DESTINATION "${GARGLKINI}")
```

Comment:
Instead of `${GARGLKINI}` this should be `${CMAKE_INSTALL_SYSCONFDIR}`.  Otherwise it will think the directory is called `/etc/garglk.ini` and install a file `/etc/garglk.ini/garglk.ini`.

But then there's a disconnect (and that disconnect is why you used `${GARGLKINI}`, I'm sure): `${GARGLKINI}` is guaranteed to have the right value, so it should be used directly for installing `garglk.ini`.

The only reason `GARGLKINI` can be specified is because the old Jam build system supported it, so I carried it over.  But there's no real reason to allow an arbitrary file as the default config.  So my (last!) request is this:

Always set `GARGLKINI` based on `${CMAKE_INSTALL_SYSCONFDIR}`: don't make it a user option.  This should be set unconditionally: whether or not the user wants to _install_ the config file, it should always know to look for it in the sysconf dir.

If `INSTALL_GARGLKINI` is set, then install to `${CMAKE_INSTALL_SYSCONFDIR}`.

If the user wants a different directory, `-DCMAKE_INSTALL_SYSCONFDIR` can be used to set it.  If the user wants a different name than `garglk.ini`, that's not going to be possible.

Thanks for working on this!